### ISSUE TITLE
Elliptic inverse 2D

### ISSUE THREAD

**Issue by ediemiglio** [lululxvi/deepxde#255]
Dear Lu,
   I am trying to solve a 2D counterpart of the example "elliptic_inverse_field.py". I have modified your code as follows but it does not work and I am not able to undestand what is the problem:

`import deepxde as dde
import matplotlib.pyplot as plt
import numpy as np
from deepxde.backend import tf

def main():
    def pde(x, y):
        u, q = y[:, 0:1], y[:, 1:2]
        du = tf.gradients(u, x)[0]
        du_x, du_y = du[:, 0:1], du[:, 1:2]
        du_xx = tf.gradients(du_x, x)[0][:, 0:1]
        du_yy = tf.gradients(du_y, x)[0][:, 1:2]

        return -du_xx-du_yy+q

    def sol(x):
        return np.sin(np.pi*x[:,0:1])*np.sin(np.pi*x[:,1:2])

    geom = dde.geometry.Rectangle([0,0], [1,1])

    ob_x = geom.random_points(30)
    ob_u = sol(ob_x)
    observe_u = dde.PointSetBC(ob_x, ob_u)

    data = dde.data.PDE(
        geom,
        pde,
        observe_u,
        num_domain=200,
        num_boundary=2,
        anchors=ob_x,
        num_test=1000,
    )

    net = dde.maps.FNN([2]+[20]*4+[1], "tanh", "Glorot uniform")

    model = dde.Model(data, net)
    model.compile("adam", lr=0.0001, loss_weights=[1, 100, 1000])

    losshistory, train_state = model.train(epochs=20000)
    dde.saveplot(losshistory, train_state, issave=True, isplot=False)

if __name__ == "__main__":
    main()`

Thanks in advance for your help!
   Edie

**Comment by lululxvi** [comment]
The network is not correct. It has two outputs, not one.
